generated/ruby/grpc-google-iam-v1/grpc-google-iam-v1.gemspec

```diff
@@ -0,0 +1,26 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |s|
+  s.name          = 'grpc-google-iam-v1'
+  s.version       = '0.6.8'
```

Comment:
Once we're 1.0, it's embarrassing to push breaking changes, as I did a couple weeks ago with Python `googleapis-common-protos 1.3.x` while adding LRO gRPC code.